html/changelogs/coldcola.yml

```diff
@@ -0,0 +1,4 @@
+author: coldcola
+delete-after: True
+changes:
+- rscadd: Added an IAA-only antag item : the explosive commendation poster.
```

Comment:
The : is breaking this, either do an in-body changelog, or put "" around the message..
`- rscadd: "Added an IAA-only antag item : the explosive commendation poster."`